operate/client/src/modules/components/SortableTable/index.tsx

```diff
@@ -166,41 +173,64 @@ const SortableTable: React.FC<Props> = ({
                   <tbody aria-live="polite" data-testid="data-list">
                     {rows.map((row) => {
                       const isSelected = checkIsRowSelected?.(row.id) ?? false;
+                      const operationErrorMessage =
+                        rowOperationError?.(row.id) || null;
+
+                      const {batchOperationId} =
+                        getProcessInstancesRequestFilters();
+
+                      const expandRowStyleClasses = () => {
+                        if (operationErrorMessage) {
+                          return 'errorRow';
+                        } else if (!operationErrorMessage && batchOperationId) {
+                          return 'successRow';
+                        } else {
+                          return '';
+                        }
+                      };
```

Comment:
@pedesen Since I was already using this util on the store to get the filter value I decided to just return it alongside the error message, that way the only thing depending on it is the store itself.